README.md

```diff
@@ -16,7 +16,7 @@ Yet another library to generate OpenAPI documents and validate requests & respon
 * Validate query, JSON data, response data with [pydantic](https://github.com/samuelcolvin/pydantic/) :wink:
 * Current support:
   * Flask [demo](#flask)
-  * Falcon [demo](#falcon) (including ASGI under Falcon 3+)
+  * Falcon [demo](#falcon) (including ASGI)
```

Comment:
I think we can delete (includeing ASGI) here since we requires falcon > 3.0.0

Reply:
done